crates/sui-core/src/execution_engine.rs

```diff
@@ -340,6 +340,7 @@ fn execute_transaction<S: BackingPackageStore + ParentSync + ChildObjectResolver
     (cost_summary, result)
 }
 
+#[instrument(name = "exec_engine_transfer_object", level = "debug", skip_all)]
```

Comment:
@velvia 
I believe this function should be very quick. It's entire in Rust and does not touch the VM.
Is the overhead from instrumenting too much?

Reply:
Hm, ok we can skip instrumenting this for now.